doc/mobile.md

```diff
@@ -1,13 +1,14 @@
 # Jitsi Meet apps for Android and iOS
 
-Jitsi Meet can also be built as a standalone app for Android or iOS. It uses the
+Jitsi Meet can be built as a standalone app for Android or iOS. It uses the
 [React Native] framework.
 
+**If youi want to rebuild the SDK yourself look in [Android README] or [iOS README].**
+
 First make sure the [React Native dependencies] are installed.
 
 **NOTE**: This document assumes the app is being built on a macOS system.
-
-**NOTE**: Node 6.X and npm 3.X are recommended for building.
+**NOTE**: Node 10.X and npm 6.X are recommended for building.
```

Comment:
You dont want to bump package.json too? Maybe refer to package .json in the readme?

Reply:
I updated the root README.